src/main/java/propertyFiles/Bundle.properties

```
@@ -781,7 +781,8 @@ notification.email.createDataverse=Your new dataverse named {0} (view at {1} ) w
 # Bundle file editors, please note that "notification.email.createDataset" is used in a unit test
 notification.email.createDataset=Your new dataset named {0} (view at {1} ) was created in {2} (view at {3} ). To learn more about what you can do with a dataset, check out the Dataset Management - User Guide at {4}/{5}/user/dataset-management.html .
 notification.email.wasSubmittedForReview={0} (view at {1} ) was submitted for review to be published in {2} (view at {3} ). Don''t forget to publish it or send it back to the contributor, {4} ({5})\!
-notification.email.wasReturnedByReviewer={0} (view at {1} ) was returned by the curator of {2} (view at {3} ).
+notification.email.wasReturnedByReviewer={0} (view at {1} ) was returned by the curator of {2} (view at {3} ).{4}
+notification.email.wasReturnedByReviewerReason=See the curator comment : {0}
```

```suggestion
notification.email.wasReturnedByReviewerReason=See the curator comment: {0}
```

I think "curator" is ok since we use the term elsewhere ("was returned by the curator").

I'm just suggesting we remove a space before the colon.

---

Also, from @sbarbosadataverse:

'in the email reply where the reason for return is added, can we add a note that says " navigate to your dataset to edit your content?"'

---

> Also, from @sbarbosadataverse:
> 
> 'in the email reply where the reason for return is added, can we add a note that says " navigate to your dataset to edit your content?"'

What do you think about email content @DS-INRA ?